rxandroid-framework/src/main/java/rx/android/app/ReactiveDialogResult.java

```diff
@@ -0,0 +1,33 @@
+package rx.android.app;
+
+/**
+ * A wrapper for returning value from ReactiveDialog dialog.
+ *
+ * @param <V> The type of data expected as return value from the dialog.
+ */
+public final class ReactiveDialogResult<V> {
```

Comment:
This should have `equals()` and `hashCode()` implementations. A `toString()` would also be nice.